components/ConvertSteps/Converter.js

```diff
@@ -0,0 +1,195 @@
+import React, { useCallback, useEffect, useState } from 'react'
```

Comment:
Nitpick: should we rename this file `ConvertSteps` to match the directory as it's the "main" export?

Reply:
Def, I rushed this name change to fix my build issues quickly.